packages/oauth2/src/server/index.tsx

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+import express from 'express'
+import * as path from 'path'
+import {Provider} from 'oidc-provider'
+import {MongoDBAdapter as OAuth2DBAdapter} from './adapter'
+import {routes} from './routes'
+import {configuration} from './configuration'
+import url from 'url'
+import {MongoDBAdapter as WepublishDBAdapter} from '@wepublish/api-db-mongodb'
+import set from 'lodash/set'
+
+import * as jwks_keys from './keys/keys.json'
```

Comment:
Get keys from environment so it can be easily set for development or inside docker containers.